modules/dxp/apps/portal-security-audit/portal-security-audit-web/src/main/resources/META-INF/resources/configuration/icon/export_audit_events.jsp

```diff
@@ -7,7 +7,20 @@
 
 <%@ include file="/init.jsp" %>
 
-<liferay-portlet:resourceURL id="/audit/export_audit_events" var="exportURL" />
+<liferay-portlet:resourceURL id="/audit/export_audit_events" var="exportURL">
+	<portlet:param name="endDateAmPm" value="<%= String.valueOf(auditDisplayContext.getEndDateAmPm()) %>" />
```

Comment:
Do we need to use `String.valueOf` for these params, or are `int` values ok to use?

Reply:
We need String.valueOf or we get a `int cannot be converted to java.lang.String`